samples/apps/copilot-chat-app/webapp/src/App.tsx

```diff
@@ -90,6 +91,14 @@ const App: FC = () => {
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [instance, inProgress, isAuthenticated, appState]);
 
+    useEffect(() => {
+        if (isAuthenticated && account) {
+            dispatch(setLoggedInUserId(account.homeAccountId))
+        } else {
+            dispatch(setLoggedInUserId(''))
```

Comment:
My original thinking was that this would only be triggered when the user logs out. 
Since the useEffect hook is attached to isAuthenticated, at what point will isAuthenticated changes if the user is not logging out?